tools/hangar/__snapshots__/test_corpus/website.w_test_sim.md

```diff
@@ -0,0 +1,10 @@
+# [website.w](../../../../examples/tests/valid/website.w) | test | sim
+
+## stdout.log
```

Comment:
Can we also start snapshotting the simulator.json files? I think tracking these would help us detect breaking changes to the console

(maybe this should be a separate issue...)

Reply:
added: #2427